docs/observers.rst

```diff
@@ -772,6 +776,48 @@ You simply need to initialize it with your project name and (optionally) api tok
     To make things simple you can put `export NEPTUNE_API_TOKEN=YOUR_LONG_API_TOKEN`
     line to your `~/.bashrc` or `~/.bash_profile` files.
 
+.. _queue_observer:
+
+Queue Observer
+==============
+
+The `QueueObserver` can be used on top of other existing observers.
+It runs in a background thread. Observed events
+are buffered in a queue and the background thread is woken up to process
+new events at a fixed interval of 20 seconds be default.
+If the processing of an event fails, the event is put back on the queue
+and processed next time. The `QueueObserver` has currently no way
+of declaring an event as finally failed, so if the failure is not
+due to a temporary unavailability of an external service, the observer
+will try forever.
```

Comment:
Could you add one concrete use case? We don't want readers to be confused and use it in very simple cases when a simple observer would do the job.